tools/collect-mod-repro-bundle.ps1

```
@@ -0,0 +1,428 @@
+
+    $pythonCmd = @(Resolve-PythonCommand)
+    if ($pythonCmd.Count -eq 0) {
+        return [PSCustomObject]@{
+            profileId = $null
+            reasonCode = "python_not_found"
+            confidence = 0.0
+            launchKind = "Unknown"
+        }
+    }
+
+    $args = @()
```

PSScriptAnalyzer (1.23.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)